integrated_channels/blackboard/admin/__init__.py

```diff
@@ -75,3 +77,38 @@ def customer_oauth_authorization_url(self, obj):
             return format_html((f'<a href="{obj.oauth_authorization_url}">Authorize Link</a>'))
         else:
             return None
+
+
+@admin.register(BlackboardLearnerDataTransmissionAudit)
+class BlackboardLearnerDataTransmissionAuditAdmin(admin.ModelAdmin):
+    """
+    Django admin model for BlackboardLearnerDataTransmissionAudit.
+    """
+    list_display = (
+        "enterprise_course_enrollment_id",
+        "course_id",
+        "status",
+        "modified",
+    )
+
+    readonly_fields = (
+        "blackboard_user_email",
+        "progress_status",
+        "content_title",
+        "enterprise_customer_name",
+    )
+
+    list_per_page = 1000
+
+    class Meta:
+        model = BlackboardLearnerDataTransmissionAudit
+
+    def enterprise_customer_name(self, obj):
```

Comment:
This method is replicated across the AuditAdmin classes.  Could we put this method into a base class?